### PR TITLE
Fix missing quotation &  Error string should not be capitalized or end with punctuation.

### PR DESCRIPTION
**Pull request by wayilau**
This PR is a 
 - [ ] Feature Implementation
 - [x] Bug Fix
 - [ ] Documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** 

Fixes #

Please leave this checklist in the PR comment so that maintainers can ensure a good PR.

Merge Checklist:
 - [ ] New feature 
   - [ ] Tests
   - [ ] Documentation
 - [ ] SVCat CLI flag
 - [ ] Server Flag for config
   - [ ] Chart changes
   - [ ] removing a flag by marking deprecated and hiding to avoid
         breaking the chart release and existing clients who provide a
         flag that will get an error when they try to update

### CODE REVIEW

File: pkg/controller/controller_instance.go [hunk 1]
```diff
@@ -1192,7 +1192,7 @@ func (c *controller) resolveReferences(instance *v1beta1.ServiceInstance) (bool,
 		return c.resolveNamespacedReferences(instance)
 	}
 
-	return false, stderrors.New(errorAmbiguousPlanReferenceScope)
+	return false, fmt.Errorf("%v", errorAmbiguousPlanReferenceScope)
```
Comment (luksa) [inline pkg/controller/controller_instance.go]: What this change?

Comment (wayilau) [reply pkg/controller/controller_instance.go]: the content variable errorAmbiguousPlanReferenceScope is "Couldn't determine if the instance refers to a Cluster or Namespaced ServiceClass/Plan". golint says that "Error string should not be capitalized or end with punctuation".

Comment (luksa) [reply pkg/controller/controller_instance.go]: Hmm, doesn't this change then just hide it from the linter? Why don't you simply change the error message to lowercase?
And what's odd is that golint on my machine doesn't complain at all.

Comment (wayilau) [reply pkg/controller/controller_instance.go]: i saw the warning in goland, when i open it in vscode, the warning missed. i was confused. 
so i change the contenct to lowercase.

**Pull request merged**